src/main/java/com/squareup/protoparser/ProtoParser.java

```diff
@@ -401,8 +401,8 @@ private OptionElement readOption(char keyValueSeparator) {
     }
     Object value = readValue();
     Object valueOrSubOption =
-        subName != null ? OptionElement.create(subName, value, isParenthesized) : value;
-    return OptionElement.create(name, valueOrSubOption, isParenthesized);
```

Comment:
It's used, you just don't see them in this PR. There's three tests which verify explicit, parenthetical behavior.

For example:
```java
  @Test public void nestedToString() {
    OptionElement option =
        OptionElement.create("foo.boo", OptionElement.create("bar", "baz"), true);
    String expected = "(foo.boo).bar = \"baz\"";
    assertThat(option.toString()).isEqualTo(expected);
  }
```